src/pages/[domain]/admin/configure-place.tsx

```diff
@@ -1,15 +1,246 @@
+    logo: builderPlaceData?.logo || '',
+    palette,
+  };
+
+  useEffect(() => {
+    let timeoutId: string | number | NodeJS.Timeout | undefined;
+    // Prevents max stack depth being called
+    const delayedEffect = () => {
+      setPalette(prevPalette => {
+        return { ...prevPalette, [colorName]: color.hex };
+      });
+    };
+
+    timeoutId = setTimeout(delayedEffect, 10);
+    return () => clearTimeout(timeoutId);
+  }, [color]);
+
+  useEffect(() => {
+    document.documentElement.style.setProperty('--primary', palette.primary);
+    document.documentElement.style.setProperty('--primary-50', palette.primary + '60');
+    document.documentElement.style.setProperty('--primary-focus', palette.primaryFocus);
+    document.documentElement.style.setProperty('--primary-content', palette.primaryContent);
+    document.documentElement.style.setProperty('--base-100', palette.base100);
+    document.documentElement.style.setProperty('--base-200', palette.base200);
+    document.documentElement.style.setProperty('--base-300', palette.base300);
+    document.documentElement.style.setProperty('--base-content', palette.baseContent);
+    document.documentElement.style.setProperty('--info', palette.info);
+    document.documentElement.style.setProperty('--info-content', palette.infoContent);
+    document.documentElement.style.setProperty('--success', palette.success);
+    document.documentElement.style.setProperty('--success-50', palette.success + '60');
+    document.documentElement.style.setProperty('--success-content', palette.successContent);
+    document.documentElement.style.setProperty('--warning', palette.warning);
+    document.documentElement.style.setProperty('--warning-content', palette.warningContent);
+    document.documentElement.style.setProperty('--error', palette.error);
+    document.documentElement.style.setProperty('--error-content', palette.errorContent);
+  }, [palette]);
+
+  if (loading) {
+    console.log('no data');
+    return (
+      <div className='flex flex-col mt-5 pb-8'>
+        <Loading />
+      </div>
+    );
+  }
+
+  if (!builderPlaceData) {
+    return (
+      <div className='flex flex-col mt-5 pb-8'>
+        <p>No builderPlace found in link to this wallet</p>
+      </div>
+    );
+  }
+
+  const handleSubmit = async (
+    values: IFormValues,
+    { setSubmitting }: { setSubmitting: (isSubmitting: boolean) => void },
+  ) => {
+    if (walletClient && account?.address) {
+      setSubmitting(true);
+      try {
+        /**
+         * @dev Sign message to prove ownership of the address
+         */
+        const signature = await walletClient.signMessage({
+          account: account.address,
+          message: values.subdomain,
+        });
+
+        await updateBuilderPlaceAsync({
+          subdomain: values.subdomain,
+          logo: values.logo,
+          name: builderPlaceData.name,
+          ownerTalentLayerId: builderPlaceData.ownerTalentLayerId,
+          palette,
```

Comment:
do you have it updated in the database after submitting ?

Reply:
Tested and worked for me